test/test_jsonplaceholder.py

```diff
@@ -174,6 +179,28 @@ def test_create_post_accesses_the_right_endpoint_when_called(self):
                 url="https://jsonplaceholder.typicode.com/posts",
                 params={},
                 json={"title": title, "body": content, "userId": user_id},
+                files=[],
                 headers={"Content-type": "application/json; charset=UTF-8"},
             )
             self.assertIs(api.create_post.response, response)
+
+    def test_upload_file_accesses_the_right_endpoint_when_called(self):
+        api = qrest.API(jsonplaceholderconfig)
+        api.upload_file.response = ContentResponse()
+
+        file = open(qrest.__file__, 'rb')
```

Comment:
Extreme nitpick: the test doesn't close the open file. This should only be a theoretical problem because the file will be closed when the last reference to its object is deleted.

Reply:
Agreed, missed this one.